.github/workflows/build_and_deploy_downloader.yml

```
@@ -55,11 +55,11 @@ jobs:
 
           - name: download-osx-x64
             target: x86_64-apple-darwin
-            os: macos-13
+            os: macos-15-intel
```

ここ、なんででしたっけ？普通にクロスコンパイルできそうな気がしています。

```suggestion
            os: macos-14
```